pkg/cluster/kubernetes/resource/load_test.go

```diff
@@ -263,13 +264,57 @@ func debyte(r resource.Resource) resource.Resource {
 	return r
 }
 
+func TestSoftDecryption(t *testing.T){
```

Comment:
I don't think this test adds a lot of value. AFAICT it only tests the internal strategy mechanism.

Reply:
Yeah. The idea was to avoid testing sops functionality in a unit test, since it should be covered by the sops test suite, but I get what you mean.